db/migrations/20201028151313_initial.js

```diff
@@ -10,7 +10,6 @@ exports.up = function(knex) {
       table.string('number_of_players');
       table.string('video');
       table.boolean('family_friendly').defaultTo(1);
-      table.timestamps(true, true);
```

Comment:
I think it was a good decision to not use `timestamps` for the scope of this project but I do want to learn how to use it. I think  it would be very helpful for the `post`methods

Reply:
I agree, Orlando!